numato_relay/numato_relay/numato_relay.py

```diff
@@ -38,7 +38,7 @@
 class NumatoRelay(Node):
     def __init__(self):
         super().__init__('NumatoRelay')
-        self.service_set_relay = self.create_service(SetRelay, '/set_relay', self.set_relay)
+        self.service_set_relay = self.create_service(SetRelay, 'set_relay', self.set_relay)
```

Comment:
Not necessarily a blocker, but why use one service with a custom message instead of multiple services that use the `std_srvs/SetBool` type?

Reply:
I've considered it, but I am keeping this as a common service since it matches the ROS 1 implementeation.

Reply:
The ROS 1 implementation uses multiple `std_srvs/SetBool` services, not a single common service

See: https://github.com/clearpathrobotics/numato_relay/blob/main/scripts/numato_relay_interface#L47-L54